### DISCUSSION TITLE
Sorting by the "d" Attribute

### DISCUSSION THREAD

**Discussion by pknowl** [the-human-colossus-foundation/oca-spec#71] [General]
What was the rationale behind sorting overlays by the "d" attribute instead of the overlay name, which is more intuitive for humans to read and the first thing available for sorting?

**Comment by blelump** [comment]
@pknowl please ask the question in the former discussion https://github.com/the-human-colossus-foundation/oca-spec/discussions/69 . It ensures the reasoning about particular topic has full provenance in one place.

**Comment by pknowl** [comment]
Close discussion. It has been integrated into #69.
